### DISCUSSION TITLE
Custom managed property don't return any data

### DISCUSSION THREAD

**Discussion by nda1974** [microsoft-search/pnp-modern-search#3420] [Q&A] [answered]
I'm using pnp-modern-search v4 and have created my own managed property and mapped it to a list column. However when I look at the Debug layout
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8806211/24a09da1-87db-4a40-ba21-8b92032073db)
the properties returned doesn't contain any value.
When I use the good old SharePoint Search Query Tool
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8806211/9f86164e-c139-4543-bc4f-8e5d918ce24d)
the data is returned as expected.
Any help is appreciated :-)

**Comment by kasperbolarsen** [comment]
Are you using SP or MS search as source?

Are you 100% sure the query is identical?

Is fieldPostnummerby the same as fieldPostNrBy?
Can it be a typo?

( perhaps use ZipCode in order for it to make sense for somebody not from Scandinavia 😉)

**Reply by nda1974** [reply]
Hi Kasper,
i'm using SP as a search source. 
It's a little difficult to tell if the query is identical. I can only see the full query url from SharePoint Search Tool but not from the pnp-modern-search webpart. The query text however is identical and i have made sure to select the properties in the webpart property pane.
(and you're right about the naming :-) )

**Reply by kasperbolarsen** [reply, marked as answer]
Have a look at https://youtu.be/kPMSiRxK8H8 perhaps it will give you the info you need to pinpoint the issue

**Reply by nda1974** [reply]
Thank you for a great tip :-)
I found that the problem was that when testing my webpart in the workbench the search URL was pointing to another sitecollection which usually aint a problem but in this case I got null values for columns that was defined as sitecolumns and that's the reason why search can't return any values
